tests/test_sanity.py

```diff
@@ -329,8 +326,6 @@ def test_llvm_fastcomp(self):
 
     restore_and_set_up()
 
-    self.check_working([EMCC] + MINIMAL_HELLO_WORLD + ['-s', 'ASM_JS=0'], '''Compiler settings are incompatible with fastcomp. You can fall back to the older compiler core, although that is not recommended''')
-
```

Comment:
If you are going to remove this test, why not also remove the error message in emcc.py that cause it.  Better still, revert this line and make a separate CL for removing that.

Reply:
Ok.